src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/FormattingHelpers.cs

```diff
@@ -431,6 +431,7 @@ public static bool IsInitializerForArrayOrCollectionCreationExpression([NotNullW
             {
                 var parent = initializer.Parent;
                 if (parent is ArrayCreationExpressionSyntax ||
```

Comment:
@sharwell would it just be simpler to suppress if the kind of the initializer was a connection initializer?